app/src/main/kotlin/com/intive/picover/common/validator/TextValidator.kt

```diff
@@ -8,24 +8,23 @@ import com.intive.picover.common.validator.ValidationStatus.ValidText
 class TextValidator private constructor(
 	private val allowEmpty: Boolean,
 	private val allowBlank: Boolean,
-	private val maxLength: Int?,
 ) {
 
-	fun validatingText(text: String) =
-		when {
-			!allowEmpty && text.isEmpty() -> EmptyText
-			!allowBlank && text.isBlank() -> BlankText
-			maxLength != null && text.length > maxLength -> TooLongText
-			else -> ValidText
-		}
+	fun validatingText(
+		text: String,
+		isLongText: Boolean = false,
+	) = when {
+		!allowEmpty && text.isEmpty() -> EmptyText
+		!allowBlank && text.isBlank() -> BlankText
+		text.length > if (isLongText) 80 else 20 -> TooLongText
+		else -> ValidText
+	}
 
 	class Builder {
 		var allowEmpty = true
 		var allowBlank = true
-		var maxLength: Int? = null
 
-		fun build() =
-			TextValidator(allowEmpty, allowBlank, maxLength)
+		fun build() = TextValidator(allowEmpty, allowBlank)
```

Comment:
It should be left as it was before, it was automatically changed during ktlint bump.

Reply:
Updated